airship.cabal

```diff
@@ -81,6 +81,7 @@ test-suite unit
                , tasty            >= 0.10.1  && < 0.12
                , tasty-quickcheck >= 0.8.3   && < 0.8.5
                , tasty-hunit        >= 0.9.1 && < 0.10
+               , hspec
```

Comment:
Whoops. Was going to add tests, realized the unexposed Internal modules make it a PITA to test, backed out with plans to return to it.